src/app/AuthenticationController.tsx

```diff
@@ -257,6 +259,30 @@ class AuthenticationController extends React.Component<AuthenticationControllerP
     }
   }
 
+  // Check which clusters does not have an accessible Kiali instance.
+  // Emit a warning telling that for those clusters, no cross-links will be available.
+  private checkConfiguredRemoteKialis(backendConfigs: ServerConfig) {
+    if (backendConfigs.clusters) {
+      const clustersWithoutKialis = [] as string[];
+      for (let cluster in backendConfigs.clusters) {
+        if (backendConfigs.clusters.hasOwnProperty(cluster)) {
+          const kialiInstance = backendConfigs.clusters[cluster].kialiInstances?.find(instance => instance.url.length !== 0);
+          if (!kialiInstance) {
+            clustersWithoutKialis.push(cluster);
+          }
+        }
+      }
+
+      if (clustersWithoutKialis.length > 0) {
+        AlertUtils.addWarning(
+          "Not all remote clusters have reachable Kiali instances.",
+          undefined, undefined,
+          "Context menus are diasabled for remote cluster nodes without a discovered Kiali or if it is not configured for remote links."
```

Comment:
diasabled -> disabled

Actually, a slight recommended change:

Context menus are disabled for remote cluster nodes if a Kiali instance is not discovered, or if the remote Kiali is not configured with an external URL.

Reply:
It's now fixed.